context/microfrontends.tsx

```diff
@@ -2,34 +2,52 @@ import { ExampleTinyFrontendType } from "@tiny-frontend/example-tiny-frontend-co
 import React, { PropsWithChildren, ReactNode } from "react";
 
 export interface MicroFrontendStore {
-  ExampleTinyFrontendServer: ExampleTinyFrontendType;
+  ExampleTinyFrontendServer: ExampleTinyFrontendType | string;
 }
 
 export const initialState: MicroFrontendStore = {
-  ExampleTinyFrontendServer: () => null,
+  ExampleTinyFrontendServer: () => null
 };
 
 const MicrofrontendsContext =
   React.createContext<MicroFrontendStore>(initialState);
 
-const MicrofrontendsContextProvider: React.FC<PropsWithChildren<any>> = ({
+const getMicrofrontends = (values: MicroFrontendStore): MicroFrontendStore =>
+  Object.keys(values).reduce(
+    (acc, curr) => {
+      if (
+        // ignore this hard check
+        curr === "ExampleTinyFrontendServer" &&
+        typeof values.ExampleTinyFrontendServer === "string"
+      ) {
+        const fnWorking = "return function () { return `Hello, world` }";
+        const [args, ...rest] = values.ExampleTinyFrontendServer.split("=>");
+        const fn = `return function ${curr}${args} ${rest.join("=>")}`;
+        console.log({ fn });
+
+        return {
+          ...acc,
+          ExampleTinyFrontendServer: new Function(fn)() // if we replace with fnWorking, it works
```

Comment:
This one also returns a function, but it throws a ReferenceError to `t`, which stands for React. 

Do you know what can be behind this @adrienbaron ?